### PR TITLE
Story 1 into Main - Application Architecture

### PR DESCRIPTION
**Pull request by SniffleSneeze**
- Wireframe 
- Data structure

### CODE REVIEW

File: css/normalize.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (mporam) [inline css/normalize.css]: why? remove this

File: index.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (mporam) [inline index.php]: please remove this file

File: details.php [hunk 3]
```diff
@@ -0,0 +1 @@
+<?php
```
Comment (mporam) [inline details.php]: remove this file

File: utilities/requestJson.php [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?php
```
Comment (mporam) [inline utilities/requestJson.php]: remove this file

**Review by mporam — CHANGES_REQUESTED** [review]
remove the big gap on the car details page

**Review by nealioa — DISMISSED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**